setup.py

```diff
@@ -135,7 +135,8 @@
             'pddf_ledutil = pddf_ledutil.main:cli',
             'show = show.main:cli',
             'sonic-clear = clear.main:cli',
-            'sonic_installer = sonic_installer.main:cli',
+            'sonic-installer = sonic_installer.main:sonic_installer',
+            'sonic_installer = sonic_installer.main:sonic_installer',
```

Comment:
How about adding a depreciation warning error message to stderr if the underscore binary is called?

Reply:
Great idea! I added a deprecation warning in the latest commit for the root command, but I think it would also be good to do it for the subcommands, as well.